guacamole-docker/README.md

```diff
@@ -148,6 +145,24 @@ binding as `LDAP_SEARCH_BIND_DN` to authenticate other users. (Optional)
 username within all Guacamole user objects in the LDAP directory. (Optional)
 9. `LDAP_CONFIG_BASE_DN` - The base of the DN for all Guacamole configurations. (Optional)
 
+Deploying Apache Guacamole with LDAP authentication
```

Comment:
What do you think about letting the "Using both LDAP and a database" thing be a level 3 heading beneath the general LDAP section? Like what we did with the database initialization parts?

Seems related enough that it maybe shouldn't be its own thing.